packages/lesswrong/server/sqlConnection.ts

```diff
@@ -167,8 +167,13 @@ const logIfSlow = async <T>(
 
   const milliseconds = endTime - startTime;
   if (logAllQueries) {
-    // eslint-disable-next-line no-console
-    console.log(`Finished query #${queryID} (${milliseconds} ms)`);
+    if (logResultSetSizes) {
```

Comment:
Does this really need its own setting or could we just include this info whenever `logAllQueries` is enabled (since it's a strict superset)?

I guess there's a performance implication to stringifying every query result, but I figure that probably doesn't really matter if `logAllQueries` is enabled (the syscall to print will take longer than stringifying anyway).